src/games/chess.h

```diff
@@ -18,8 +18,8 @@ namespace chess {
 
 struct ChessBoard {
 
-  static const size_t boardSize = 8;
-  static const size_t boardDim = boardSize + 4;
+  static const int boardSize = 8;
```

Comment:
shouldn't these be uints?

Reply:
Not necessarily, it's just 8, which can be any data type. I bet it just gets optimized.

It gets assigned to `_featSize` which is `vector<int64_t>` below.